packages/core/src/Editor.ts

```diff
@@ -773,4 +782,34 @@ export class Editor extends EventEmitter<EditorEvents> {
   get $doc() {
     return this.$pos(0)
   }
+
+  /**
+   * Returns a set of utilities for working with positions and ranges. These
+   * utilities let you calculate the new position or range after applying a
+   * transaction.
+   *
+   * @returns The position helpers.
+   */
+  public get positionHelpers(): PositionHelpers {
```

Comment:
Does this make sense as a getter on the Editor class? I thought this would just be utility functions I important and pass my props through to map abs/rel positions. Why doe we need it here?

Also why do the positionHelpers exist on the collaboration state?